src/wallet/rpcwallet.cpp

```diff
@@ -3892,6 +3892,7 @@ UniValue z_gettotalbalance(const UniValue& params, bool fHelp)
     if (fHelp || params.size() > 2)
         throw runtime_error(
             "z_gettotalbalance ( minconf includeWatchonly )\n"
+            "\nDEPRECATED\n"
```

Comment:
Please add a suggestion to use `z_getbalanceforaccount` (if I'm remembering the name correctly) instead.